### PR TITLE
Added ThemesBerg to Themes&Templates under Frontend Section

### LINKED ISSUE #2027: [Add] New Link to Themes in Front-End.
### Site URL

https://themesberg.com/

### Description of the above provided site

It is a website for free and premium Bootstrap based Templates, Themes and built-in UI Tool kits.
**Free and Premium** themes, templates, admin dashboards and UI kits built in Bootstrap, React, Laravel, and Figma used by many popular web developers and software companies

### Category

Frontend

### Sub Category

"themes & templates"

### Checklist

- [X] I have checked the existing issues
- [X] I have read the [Contributing Guidelines](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md)
- [X] I am willing to work on this issue (optional)
- [X] I am participating in Hacktoberfest 2023

### PR DESCRIPTION
**Pull request by surya-mu**
## Fixes Issue 
Closes #2027 

## Changes proposed

Added Themesberg to Themes&Templates section under Frontend Section.

## Note to reviewers

I am open to any suggestions regarding this PR.

### CODE REVIEW

File: database/frontend/themes-templates.json [hunk 1]
```diff
@@ -54,5 +54,12 @@
     "url": "https://freefrontend.com/",
     "category": "frontend",
     "subcategory": "themes-templates"
+  },
+  {
+    "name": "ThemesBerg",
+    "description": "Provides Free and Premium Templates, Themes from a wide variety of Web Developers and Software Companies",
```
Comment (CBID2) [inline database/frontend/themes-templates.json:60]:
```suggestion
    "description": "Provides free and premium templates and themes from a wide variety of web development and software companies",
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Hi @surya-mu! 😊 Your description could be improved. Other than that, you're off to a great start! 😊

**Review by Anmol-Baranwal — APPROVED** [review]
🚀 Looks good to me.

**Review by CBID2 — APPROVED** [review]
Looks great to me! @rupali-codes, you're up!

**Review by rupali-codes — APPROVED** [review]

**Pull request merged**